### PR TITLE
feat(web): addition of x500 alias for exchange email

### PR DESCRIPTION
**Pull request by anooparveti**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Feat #MANAGER-8731
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~ [n/a]

## Description

Support to add X500 alias for an exchange e-mail

### CODE REVIEW

**Review by ghyenne — APPROVED** [review]

File: packages/manager/modules/exchange/src/account/alias/account-alias.html [hunk 1]
```diff
@@ -13,7 +13,12 @@
             <oui-datagrid-column
                 data-title="'exchange_tab_ALIAS_header' | translate"
                 data-property="displayName"
-            ></oui-datagrid-column>
+            >
+                <div
+                    class="account-alias-column text-wrap"
+                    data-ng-bind="$value"
+                ></div>
```
Comment (mohammed-zahaf) [inline packages/manager/modules/exchange/src/account/alias/account-alias.html:20]: I think we must have the P element to display the text.
```suggestion
                <div
                    class="account-alias-column text-wrap"
                ><p data-ng-bind="$value"></p></div>
```

**Review by radireddy — APPROVED** [review]

File: packages/manager/modules/exchange/src/account/alias/add/account-alias-add.controller.js [hunk 2]
```diff
@@ -108,17 +122,25 @@ export default class ExchangeAddAccountAliasCtrl {
...
+        !isEmpty(this.model.alias) &&
+        isString(this.model.alias) &&
+        this.model.alias.length <= 64;
+      const hasDomain = !isEmpty(this.model.domain);
+      isAliasValid = aliasIsValid && hasDomain && !this.takenEmailError;
+    } else {
+      const match = this.model.x500.match(X500_REGEX) || [];
+      isAliasValid = match.length === 1 && match[0] === this.model.x500;
     }
+    return isAliasValid;
```
Comment (mohammed-zahaf) [inline packages/manager/modules/exchange/src/account/alias/add/account-alias-add.controller.js:138]: I think we could do that:

```suggestion
    
    if (this.model?.radio === this.ALIAS_TYPE.smtp) {
      const aliasIsValid =
        !isEmpty(this.model.alias) &&
        isString(this.model.alias) &&
        this.model.alias.length <= 64;
      const hasDomain = !isEmpty(this.model.domain);
      return= aliasIsValid && hasDomain && !this.takenEmailError;
    } 
     
     const match = this.model.x500.match(X500_REGEX) || [];
     return match.length === 1 && match[0] === this.model.x500;
   
```

Comment (anooparveti) [reply packages/manager/modules/exchange/src/account/alias/add/account-alias-add.controller.js:138]: I totally agree with this suggestion. But I was getting a lint error for putting statements after 1st `return`

**Review by radireddy — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by radireddy — DISMISSED** [review]

**Review by mohammed-zahaf — DISMISSED** [review]

**Comment by antleblanc** [comment]
Pulling translations.

**Comment by JeremyDec** [comment]
Hello @anooparveti some conflicts needs to be resolved before to merge this PR. Can you check please?
Thanks

**Review by JeremyDec — CHANGES_REQUESTED** [review]
Hello @anooparveti some conflicts needs to be resolved before to merge this PR. Can you check please?
Thanks

**Comment by antleblanc** [comment]
Rebasing on `develop` branch.

**Comment by antleblanc** [comment]
Rebased successfully on `develop` branch (conflicts resolved).

**Review by antleblanc — DISMISSED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**